python_bindings/test/correctness/addconstant_test.py

```diff
@@ -223,6 +223,10 @@ def test(addconstant_impl_func, offset):
     else:
         assert False, "Did not see expected exception!"
 
+    # Implicit conversions for large integers
+    assert (hl.i32(0) + 0x7fffffff).type() == hl.Int(32)
```

Comment:
`hl` isn't in scope here, and **can't** be: this is a test of AOT compilation, and `import halide as hl` is only valid for JIT or Generator code. This test should probably go into something like python_bindings/test/correctness/basics.py instead.

Reply:
whoops I moved it to that file